lib/logstash/inputs/file.rb

```diff
@@ -62,6 +62,22 @@
 # to the rotation and its reopening under the new name (an interval
 # determined by the `stat_interval` and `discover_interval` options)
 # will not get picked up.
+
+#TODO - move this to core
```

Comment:
I believe This `TODO` will be pickup by the documentation generator, might be better to remove it